build.sbt

```diff
@@ -21,7 +21,7 @@ scalacOptions ++= Seq(
   "-unchecked",                 // Enable additional warnings where generated code depends on assumptions.
   "-Wdead-code",                // Warn when dead code is identified.
   "-Wextra-implicit",           // Warn when more than one implicit parameter section is defined.
-  "-Wunused",                   // Warn if something from check list is unused.
+//  "-Wunused",                   // Warn if something from check list is unused.
```

Comment:
It's not a good idea to disable all warnings

It's better to add exceptions for quill. For example:
```scala
"-Wconf:msg=(Quoted|idiomContext@_)&cat=unused:s"
```